packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReview.css

```diff
@@ -1,12 +1,60 @@
-.text_content {
-    padding: 25px;
-    padding-top: 50px;
+.total_quantity {
     font-weight: bold;
 }
 
-.container {
-    min-height: 20rem;
+.show_all_items_button {
+    composes: root_normalPriority from '../../Button/button.css';
+    width: 100%;
+    border: none;
+}
+
+.show_all_items_button > span {
+    font-weight: bold;
+    font-size: 0.9rem;
```

Comment:
@revanth0212 - I replied to your comment in another PR, but wanted to clarify here too just in case; still use `rem` units, do not use units that result in fractional pixel values.